conf/zeppelin-site.xml.template

```diff
@@ -553,6 +553,14 @@
 </property>
 -->
 
+<!--
+<property>
+  <name>zeppelin.server.xcontent.type.options</name>
+  <value>nosniff</value>
+  <description>The HTTP X-Content-Type-Options response header helps to prevent MIME type sniffing attacks. It directs the browser to honor the type specified in the Content-Type header, rather than trying to determine the type from the content itself. The default value "nosniff" is really the only meaningful value.</description>
```

Comment:
This header is supported on all Browsers with exception to Safari and Safari on iOS. Please add this word of caution in description.

Reply:
Done!